src/main/java/org/moditect/kcctl/command/SetContextCommand.java

```diff
@@ -45,14 +52,46 @@
     @Option(names = { "--password" }, description = "Password for basic authentication")
     String password;
 
+    @Option(names = { "-o", "--client-config" }, description = "Configuration for client")
+    String[] clientConfigs;
+
+    @Option(names = { "-f", "--client-config-file" }, description = "Configuration file for client")
+    String clientConfigFile;
+
     @Override
-    public void run() {
+    public Integer call() throws FileNotFoundException, IOException {
         ConfigurationContext context = new ConfigurationContext();
-        context.setContext(contextName, new Context(URI.create(cluster), bootstrapServers, offsetTopic, username, password));
+
+        Properties clientConfigProps = new Properties();
+
+        if (!Strings.isBlank(clientConfigFile)) {
+            final String expandedPath = clientConfigFile.replaceFirst("^~", System.getProperty("user.home"));
```

Comment:
Instead of the replacement, could you try with the `Paths` API and `toAbsolutePath()`? No blocker if it doesn't work, but it somehow feels safer that way.

Reply:
Good call, I actually meant to ask about this. I like using ~, and noticed that failed, so I added this hack.  I'll take a look!

Reply:
I decided to go with your suggestion, it doesn't support ~, but I think that's fine.